templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,47 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+<h1>Manufacturer List <a style="margin-left: 20px"
+                         href="{% url 'taxi:manufacturer-create' %}">+</a>
+</h1>
+{% if manufacturer_list %}
+<table class="table">
+  <tr>
+    <th>ID</th>
+    <th>Name</th>
+    <th>Country</th>
+    <th>UPDATE</th>
+    <th>DELETE</th>
+  </tr>
 
-    {% if manufacturer_list %}
-      <table class="table">
-        <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
-        </tr>
-
-        {% for manufacturer in manufacturer_list %}
-            <tr>
-              <td>
-                  {{ manufacturer.id }}
-              </td>
-              <td>
-                  {{ manufacturer.name }}
-              </td>
-              <td>
-                  {{ manufacturer.country }}
-              </td>
-          </tr>
-        {% endfor %}
-      </table>
-
-    {% else %}
-        <p>There are no manufacturers in the service.</p>
-    {% endif %}
+  {% for manufacturer in manufacturer_list %}
+    <td>
+      {{ manufacturer.id }}
+    </td>
+    <td>
+      {{ manufacturer.name }}
+    </td>
+    <td>
+      {{ manufacturer.country }}
+    </td>
+    <td>
+      <button class="btn btn-success">
+        <a style="color: white"
+           href="{% url "taxi:manufacturer-update" pk=manufacturer.id %}"
```

Comment:
Use single quotes for the URL template tag to maintain consistency with other parts of the code. Change `"taxi:manufacturer-update"` to `'taxi:manufacturer-update'`.